assets/app/views/project.html

```diff
@@ -145,7 +145,7 @@ <h2 ng-if="!displayRouteByService[serviceId]">
             Pods in deployments created from this deployment config will be routed to by this service.
             Show things related to triggers that are about to create stuff, like builds.
             -->
-            <triggers triggers="deploymentConfig.spec.triggers"></triggers>
+            <triggers triggers="deploymentConfig.spec.triggers" builds-by-output-image="buildsByOutputImage" namespace="deploymentConfig.metadata.namespace"></triggers>
```

Comment:
since right now we only have DCs from the current project we could.  it was just that technically it should be the namespace of the deployment config, in case someone was copying things around I didn't want them putting the wrong namespace in.   I was on the fence about it :-\